x-pack/legacy/plugins/reporting/server/browsers/chromium/driver_factory/index.ts

```diff
@@ -3,39 +3,43 @@
  * or more contributor license agreements. Licensed under the Elastic License;
  * you may not use this file except in compliance with the Elastic License.
  */
+import del from 'del';
 import fs from 'fs';
 import os from 'os';
 import path from 'path';
 import {
   Browser,
-  Page,
-  LaunchOptions,
   ConsoleMessage,
+  LaunchOptions,
+  Page,
   Request as PuppeteerRequest,
 } from 'puppeteer';
-import del from 'del';
 import * as Rx from 'rxjs';
-import { ignoreElements, map, mergeMap, tap } from 'rxjs/operators';
 import { InnerSubscriber } from 'rxjs/internal/InnerSubscriber';
-
-import { BrowserConfig, NetworkPolicy } from '../../../../types';
+import { ignoreElements, map, mergeMap, tap } from 'rxjs/operators';
+import { BROWSER_TYPE } from '../../../../common/constants';
+import { ReportingConfigType } from '../../../../server/types';
+import { NetworkPolicy } from '../../../../types';
 import { LevelLogger as Logger } from '../../../lib/level_logger';
-import { HeadlessChromiumDriver } from '../driver';
 import { safeChildProcess } from '../../safe_child_process';
-import { puppeteerLaunch } from '../puppeteer';
+import { HeadlessChromiumDriver } from '../driver';
 import { getChromeLogLocation } from '../paths';
+import { puppeteerLaunch } from '../puppeteer';
 import { args } from './args';
 
 type binaryPath = string;
 type queueTimeout = number;
 
+type BrowserConfig = ReportingConfigType['capture']['browser']['chromium'];
+type ViewportConfig = ReportingConfigType['capture']['viewport'];
```

Comment:
This seems like a bug with the old type declarations: viewport is not a child of `BrowserConfig`